source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1369,17 +1398,21 @@ we distinguish two phases of connecting to a server and using it for application
 
 If there is a network error or timeout on the connection before the handshake completes,
 the client MUST replace the server's description
-with a default ServerDescription of type Unknown,
-and fill the ServerDescription's error field with useful information.
+with a default ServerDescription of type Unknown when the TopologyType is not
+LoadBalanced, and fill the ServerDescription's error field with useful information.
+
+If there is a network error or timeout on the connection before the handshake completes,
+and the TopologyType is LoadBalanced, the client MUST keep the ServerDescription
+as LoadBalancer and fill the ServerDescription's error field with useful information.
```

Comment:
Why are we modifying the description's `error` field? The server remains selectable and we don't publish `ServerDescriptionChanged` events, so the error would never be surfaced to the user as part of a server selection error or SDAM event. It seems simpler to leave the entire description as-is.

Reply:
Removed the bit about updating the error field.